helpdesk-bot/src/main/java/org/symphonyoss/symphony/bots/helpdesk/bot/listener/HelpDeskRoomEventListener.java

```diff
@@ -115,8 +132,43 @@ public void onSymUserLeftRoom(SymUserLeftRoom symUserLeftRoom) {
     SymUser symUser = symUserLeftRoom.getAffectedUser();
     SymStream symStream = symUserLeftRoom.getStream();
 
-    if (isBotUser(symUser) && isAgentStreamId(symStream)) {
-      LOGGER.warn("Bot user was removed from the agent room");
+    if (!isAgentStreamId(symStream) && (symStream.getMembers().size() <= 1)) {
+      Ticket ticket = ticketClient.getTicketByServiceStreamId(symStream.getStreamId());
+
+      if (ticket != null) {
+        LOGGER.info("Only the bot was left in the ticket room. Reopening ticket in the Agent room");
+
+        // Update ticket to a state that it can be claimed again by another agent
+        ticket.setAgent(null);
+        ticket.setState(TicketClient.TicketStateType.UNSERVICED.toString());
+        ticketClient.updateTicket(ticket);
+
+        SymUser localUser = symphonyClient.getLocalUser();
+
+        // Retrieve the client question
+        String question = StringUtils.EMPTY;
```

Comment:
I believe we might use a default text here as fallback instead of an empty string. My suggestion is: 'N/A'

Reply:
Done